README.md

```diff
@@ -178,102 +164,117 @@ app.register(RenderToken, render);
 The render token is used to register the render function with the fusion app. This is a function that knows how to
 render your application on the server/browser, and allows `fusion-core` to remain agnostic of the virtualdom library.
 
-## Plugins
+---
```

Comment:
We've generally been using line breaks for the outermost headers (`###`).  Thoughts on using them for inner headers like this as well?